SekoiaIO/endpoint/_meta/smart-descriptions.json

```diff
@@ -11,6 +11,18 @@
             }
         ]
     },
+    {
+        "value": "The agent on {host.hostname} didn't send events for the last {action.properties.time_without_events} seconds",
+        "conditions": [
+            {
+                "field": "event.action",
+                "value": "agent-inactivity"
+            },
+            {
+                "field": "action.properties.time_without_events"
+            }
```

Comment:
```suggestion
            {
                "field": "action.properties.time_without_events"
            },
            {
                "field": "host.hostname"
            }
```

Reply:
I recommended that you set `host.hostname` as a required condition

Reply:
In every event that the Agent sends the hostname is set. Not only metrics one, all of them. I'm not sure it is useful to add it in this case ...